### PR TITLE
Add array-sort blog post

### PR DESCRIPTION
**Pull request by mbasmanova**

### CODE REVIEW

**Review by majetideepak — APPROVED** [review]
nice article @mbasmanova!

**Review by xiaoxmeng — APPROVED** [review]
@mbasmanova thanks for adding this blog post!

File: website/blog/2023-07-19-array-sort.mdx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+batches. array_sort with custom comparator doesn’t lend itself well to such processing.
+
+array_sort with a transform lambda works well in a vectorized engine. To process a batch of arrays,
+Velox first evaluates the transform lambda on all the elements of the arrays, then sorts the results.
+
+
+For further reading, consider the <a href="https://opensource.googleblog.com/2022/06/Vectorized%20and%20performance%20portable%20Quicksort.html">Vectorized and performance-portable Quicksort</a>
+blog post from Google.
+
+Thank you <a href="https://www.linkedin.com/in/orrierling">Orri Erling</a> for brainstorming
```
Comment (pedroerp) [inline website/blog/2023-07-19-array-sort.mdx:202]: maybe point to their github profiles for consistency?

**Comment by pedroerp** [comment]
Small comment, but very nice read :)

**Pull request closed without merging**